### PR TITLE
Flora clean up

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR

All tree and rock prototypes are compressed from 3-6 variants into 1, that use RandomSprite

## Why / Balance

Reducing the size of prototypes. It just didn't make sense, so I fixed it.
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

### CODE REVIEW

**Comment by dvir001** [comment]
Holy based

File: Resources/migration.yml [hunk 1]
```diff
@@ -485,5 +485,55 @@ CrateCrewMonitoringBoards: CrateCrewMonitoring
...
+ShadowTree05: ShadowTree
+ShadowTree06: ShadowTree
+LightTree01: LightTree
+LightTree02: LightTree
+LightTree03: LightTree
+LightTree04: LightTree
+LightTree05: LightTree
+LightTree06: LightTree
+
+# 2024-12-16
```
Comment (slarticodefast) [inline Resources/migration.yml:538]:
```suggestion
# 2024-12-01
```
You changed the date here. You should also keep them in chronological order, so move the flora changes below the dungeon master board change.

**Review by slarticodefast — CHANGES_REQUESTED** [review]
got a minor nitpick

**Review by slarticodefast — APPROVED** [review]

**Comment by rosieposieeee** [comment]
this made it impossible to manually map all of these variants...

**Pull request merged**